File: Seminar5/HomeWork/Model/Rational/SubModelRational.java
package HomeWork.Model.Rational;

public class SubModelRational extends CalcRational {

    public SubModelRational() {

    }

    // do_it
    @Override
    public double result() {
        return x - y;
    }

    @Override
    public void setX(double value) {
        super.x = value;

    }

    @Override
    public void setY(double value) {
        super.y = value;

    }

}

File: FinalTaskCalculator/Model/Rational/SubModelRational.cs
namespace ModelRational
{

    public class SubModelRational : Rational
    {
        // do_it
        public override double Result()
        {
            return x - y;
        }

        public override void SetX(double value) => this.x = value;

        public override void SetY(double value) => this.y = value;
    }
}